apps/s_client.c

```diff
@@ -2190,6 +2193,25 @@ int s_client_main(int argc, char **argv)
                 if (strstr(mbuf, "STARTTLS"))
                     foundit = 1;
             } while (mbuf_len > 1 && mbuf[0] != '.');
+    case PROTO_SIEVE:
```

Comment:
It appears to be totally busted even if I check out. I.e. it's not this page rendering issue.